man/man8/zfs.8

```diff
@@ -1275,6 +1275,12 @@ compression algorithm compresses runs of zeros.
 This property can also be referred to by its shortened column name
 .Sy compress .
 Changing this property affects only newly-written data.
+.Pp
+When any setting except
+.Sy off
+is selected, compression will explicitly check for blocks consisting of only
+zeroes (the NUL byte).   When a zero-filled block is detected, it is stored as
```

Comment:
Please remove extra spaces before `When`.

Reply:
It was in behlendorf's suggested rewording. Will do.